Updates for Kava Ledger App

denalimarsh opened the pull request:
This PR updates the lunie ledger library to be compatible with the Kava ledger app. It updates hrp path, bip44 coin type, app name, and version.

Run tests
```bash
npm test
```

I've published it as a beta library so it can be installed in the webapp for testing
```bash
npm install @kava-labs/cosmos-ledger@0.1.0-beta.0
```

This PR is **blocked** pending integration tests. Integration testing requires the following libraries:
- Kava Ledger App: https://github.com/Kava-Labs/ledger-cosmos
- Web App: https://github.com/Kava-Labs/webapp

denalimarsh committed:
hrp path, coin type, app name, version

fedekunze commented on package.json:
```diff
@@ -1,7 +1,7 @@
-  "name": "@lunie/cosmos-ledger",
-  "version": "0.1.8",
-  "description": "provide simple Ledger tooling for the Cosmos Ledger App with user friendly errors",
+  "name": "@kava-labs/cosmos-ledger",
+  "version": "0.1.0-beta.0",
```
can we update this to be like `ledgerjs`? it's way too confusing to have 2 repos: ledger-cosmos and cosmos-ledger

nddeluca reviewed (changes requested):
Looks good, though I think we will run into errors in`ledger-cosmos-js` changing the version number to `1.0.0`.

The public key, hd path serialization, and signing chunks will end up the legacy methods - https://github.com/Zondax/ledger-cosmos-js/blob/master/src/index.js#L220.

We may want to write our own Kava JS App derived from ledger-cosmos-js to remove legacy code that isn't needed for the Kava app, ensure we can pass the same transport into multiple apps, and remove the dependence on specific cosmos app version numbers.

dependabot-preview[bot] committed:
[Security] Bump lodash from 4.17.15 to 4.17.20

Bumps [lodash](https://github.com/lodash/lodash) from 4.17.15 to 4.17.20. **This update includes security fixes.**
- [Release notes](https://github.com/lodash/lodash/releases)
- [Commits](https://github.com/lodash/lodash/compare/4.17.15...4.17.20)

Signed-off-by: dependabot-preview[bot] <support@dependabot.com>

faboweb committed:
Merge pull request #115 from luniehq/dependabot/npm_and_yarn/lodash-4.17.20

[Security] Bump lodash from 4.17.15 to 4.17.20

fedekunze committed:
update hardcoded values

fedekunze committed:
pull changes from PR #2
